doc/source/ray-air/doc_code/predictors.py

```diff
@@ -76,6 +76,32 @@ def calculate_accuracy(df):
 # Final accuracy:  0.5
 # __compute_accuracy_end__
 
+
+# __configure_batch_predictor_start__
+# Use CPUs for inference: Scaling from 1 to 4 workers, each with 3 CPUs
+predictions = batch_predictor.predict(
+    ds,
+    feature_columns=["feature_1"],
+    min_scoring_workers=1,
+    max_scoring_workers=4,
+    num_cpus_per_worker=3,
+)
+
+# Use GPUs for inference: Use 2 workers, each with 1 GPU
+predictions = batch_predictor.predict(
+    ds,
+    feature_columns=["feature_1"],
+    min_scoring_workers=2,
+    max_scoring_workers=2,
+    num_gpus_per_worker=1,
+)
+
+# Auto-scale your workers: each with 3 CPUs and 1 GPU
```

Comment:
How is this one "auto-scale" your workers vs. the other ones? Is there no cap on number of scoring workers here?

Reply:
Yes. If no min/max workers provided by user, the BatchPredictor will create an `ActorPoolStrategy` which scaling from 1 to inf.

Reply:
Agree with @justinvyu here. It's a subtle point, but the other examples are autoscaling as well. Just with different min and max values.

Reply:
I see. I'll fix it